src/basic/audit-util.c

```diff
@@ -92,8 +92,11 @@ bool use_audit(void) {
                 int fd;
 
                 fd = socket(AF_NETLINK, SOCK_RAW|SOCK_CLOEXEC|SOCK_NONBLOCK, NETLINK_AUDIT);
-                if (fd < 0)
-                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT;
+                if (fd < 0) {
+                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT && errno != EPERM;
+                        if (errno == EPERM)
+                                log_debug_errno(errno, "Audit disabled");
```

Comment:
The debug message is misleading here. EAFNOSUPPORT/EAPROTONOSUPPORT are the errors suggesting audit was actually disabled. EPERM otoh is a permission error, hence this should really say "Audit access prohibited, won't talk to audit" or something like that.